datadog_checks_base/datadog_checks/base/utils/time.py

```diff
@@ -0,0 +1,73 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import absolute_import
+
+from datetime import datetime
+from time import time as epoch_offset
+
+import pytz
+from six import PY2
+
+UTC = pytz.utc
+EPOCH = datetime(1970, 1, 1, tzinfo=UTC)
+
+
+if PY2:
+
+    def get_timestamp(dt=None):
+        """
+        Returns the number of seconds since the Unix epoch.
+        If `dt` is not specified or `None`, the current time in UTC is assumed.
+        """
+        if dt is None:
+            return epoch_offset()
+
+        return (dt - EPOCH).total_seconds()
```

Comment:
`dt` is normalized for py3 using `normalize_datetime`, any reason not to do so for py2 ?

Reply:
I thought it would use the non-None tzinfo, but you're right:

![Capture](https://user-images.githubusercontent.com/9677399/82222376-30b51180-98ef-11ea-9471-aa1bab59b4fc.PNG)